__tests__/app.test.js

```diff
@@ -221,3 +221,50 @@ describe('/api/articles/:article_id/comments', () => {
       })
     })
   })
+
+  describe('PATCH /api/articles/:article_id', () => {
+    test('PATCH /api/articles/:article_id returns a 200 status code ', () => {
+      return request(app)
+      .patch('/api/articles/1')
+      .send({ inc_votes: 1})
+      .expect(200)
+    })
+    test('PATCH /api/articles/:article_id returns the article object with an updated votes key when a positive int is used', () => {
+      return request(app)
+      .patch('/api/articles/1')
+      .send({ inc_votes: 1})
+      .expect(200)
+      .then((res) => {
+        expect(res.body).hasOwnProperty('votes')
+        expect(res.body.votes).toBe(101)
+      })
```

Comment:
You want to check the article id as well, to check that it is updating the correct article 👀